job_spec/INSAR_GAMMA_HIGH_MEMORY.yml

```
@@ -0,0 +1,102 @@
+INSAR_GAMMA:
```

Maybe? I don't actually know if this should change.
```suggestion
INSAR_GAMMA_HIGH_MEMORY:
```